### PR TITLE
Adding link bank account to settings menu

### PR DESCRIPTION
**Pull request by lisaqyli**
### Description
This PR adds link to navigate to bank account to the settings menu. The menu item is hidden behind a remote config flag `enableLinkBankAccount`. 

### Tested
Added to unit test

### CODE REVIEW

File: packages/mobile/locales/en-US/translation.json [hunk 1]
```diff
@@ -1102,5 +1102,7 @@
       "description": "You're almost done! You can connect your bank acount to Valora now via Plaid.",
       "cta": "Coming Soon!"
     }
-  }
+  },
+  "linkBankAccountSettingsTitle": "Link Bank Accounts",
+  "getAccess": "Get Access"
```
Comment (jeanregisser) [inline packages/mobile/locales/en-US/translation.json:1107]:
```suggestion
  "linkBankAccountSettingsValue": "Get Access"
```

**Review by jeanregisser — APPROVED** [review]
LGTM! Left some nits around naming.

File: packages/mobile/locales/en-US/translation.json (hunk 1 above)
Comment (lisaqyli) [reply packages/mobile/locales/en-US/translation.json]: changed the string title

File: packages/mobile/src/firebase/remoteConfigValuesDefaults.ts [hunk 2]
```diff
@@ -36,4 +36,5 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   multiTokenUseSendFlow: false,
   multiTokenUseUpdatedFeed: false,
   allowOtaTranslations: false,
+  enableLinkBankAccount: true,
```
[the remark this answers is not in the document]
Comment (lisaqyli) [reply packages/mobile/src/firebase/remoteConfigValuesDefaults.ts]: changing all instance of `enableLinkBankAccount` to `linkBankAccountEnabled`

**Comment by lisaqyli** [comment]
263386928_382543173659087_8674683976513246477_n

**Comment by ValoraQA** [comment]
Hey @MuckT,  We have verified this issue using iOS Alfa jores build V1.23.0 & updated the status in Gsheet

**Sheet Link;**
https://docs.google.com/spreadsheets/d/15Z15zu4roT6XFD1dbi9__c3Q6aQZpIiIZka1rKVrCgw/edit#gid=0

**New Bug:** [Link](https://github.com/valora-inc/wallet/issues/1630)
**Devices:** iPhone XR(14.6), iPhone 13 mini (15.1.1)
CC: @lisaqyli 
Thanks..!!

**Pull request merged**